init.sh

```diff
@@ -34,7 +37,7 @@ fi
 
 # install autossh with distro-specific commands
 
-if [[ $DISTRO = "Ubuntu" ]]; then
+if [[ ! -f initialized.txt && $DISTRO = "Ubuntu" ]]; then
```

Comment:
It's not clear to me why this is needed (ie. initialized.txt)

Reply:
So, when I was going the expect path, I was running into issues where the init.sh script was included in other scripts.  In particular the connect_* scripts both source the init.sh, but because how expect was working it was breaking the install.

At one point I added the touched initialized.txt file, so I can skip blocks that didn't need to be run a second and third time.  Ie.  The ssh key downloading and copying shouldn't be done more than once.

Since I extracted the generation of the user_config.sh the connect_* scripts just source user_config.sh instead of sourcing the init.sh.

If you like the changes I made to the connect_* scripts, then we can remove these checks.

Reply:
As discussed, let's remove the initialized.txt changes. Other changes are fine with me.